app/src/main/java/io/gnosis/safe/ui/transactions/details/view/Erc721View.kt

```diff
@@ -16,9 +17,16 @@ class Erc721View @JvmOverloads constructor(
 
     private val binding by lazy { ViewTxErc721Binding.inflate(LayoutInflater.from(context), this) }
 
-    fun setToken(logoUri: String, tokenId: String?, tokenName: String?, tokenDescription: String?) {
-        binding.tokenName.text = tokenName
+    fun setToken(logoUri: String, tokenId: String?, outgoing: Boolean, amount: String?) {
         binding.logo.loadTokenLogo(logoUri, R.drawable.ic_nft_placeholder)
         binding.tokenId.text = tokenId
```

Comment:
with(binding) { ... would look nicer

Reply:
changed in 21a7ced2831d447598816cd2a3cc59b65a8b4575